commit 79e3ee1d325bf1aea9e255127dd132265b734efe
Author: Rodrigo Freire <rfreire@rf.usersys.redhat.com>
Date:   Tue Mar 22 12:40:33 2016 -0300

    lirc_rpi: Lower IR reception error to debug
    
    Lowers a IR reception error condition message to KERNEL_DEBUG

diff --git a/drivers/staging/media/lirc/lirc_rpi.c b/drivers/staging/media/lirc/lirc_rpi.c
index cd09c99..0624439 100644
--- a/drivers/staging/media/lirc/lirc_rpi.c
+++ b/drivers/staging/media/lirc/lirc_rpi.c
@@ -271,7 +271,7 @@ static irqreturn_t irq_handler(int i, void *blah, struct pt_regs *regs)
 			data = PULSE_MASK; /* really long time */
 			if (!(signal^sense)) {
 				/* sanity check */
-				printk(KERN_WARNING LIRC_DRIVER_NAME
+				printk(KERN_DEBUG LIRC_DRIVER_NAME
 				       ": AIEEEE: %d %d %lx %lx %lx %lx\n",
 				       signal, sense, tv.tv_sec, lasttv.tv_sec,
 				       tv.tv_usec, lasttv.tv_usec);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1209_79e3ee1d325bf1aea9e255127dd132265b734efe.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1209_79e3ee1d325bf1aea9e255127dd132265b734efe.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
